pages/api/popular.ts

```diff
@@ -77,6 +77,9 @@ const PopularAssetsHandler: NextApiHandler = async (req, res) => {
       query: { name: "data product" },
     })
 
+    if (dataProductTypeIdRes.body.results[0]?.id === "undefined") {
```

Comment:
This is the one Petter suggested in the other PR?

Reply:
yes...should i put it in another PR

Reply:
It's fine by me, just curious :)